ExampleMod/Content/CustomModType/CustomModType.cs

```diff
@@ -0,0 +1,353 @@
+		}
+
+		public void Unload() {
+		}
+	}
+
+	public class HandsUpVictoryPose : ModVictoryPose
+	{
+		public override void SetStaticDefaults() {
+			VictoryPoseID.Sets.NonBoss[Type] = true;
+		}
+
+		public override void OnStartPose(Player player) {
+			if (player.whoAmI == Main.myPlayer) {
+				Main.blockMouse = true;
+			}
+		}
+
+		public override void Update(Player player) {
+			if (player.itemAnimation == 0) {
+				player.bodyFrame.Y = player.bodyFrame.Height * 5; // 2 Hands up / falling
+			}
+		}
+	}
+
+	public class HandsUpWithFireworksVictoryPose : ModVictoryPose
+	{
+		public override void SetStaticDefaults() {
+			PoseTime = 180;
+		}
+
+		public override void OnStartPose(Player player) {
+			if (player.whoAmI == Main.myPlayer) {
+				Main.blockMouse = true;
+			}
+		}
+
+		public override void Update(Player player) {
+			if (player.itemAnimation == 0) {
+				player.bodyFrame.Y = player.bodyFrame.Height * 5; // 2 Hands up / falling
+			}
+
+			if(ElapsedPoseTime == 30 || ElapsedPoseTime == 60) {
+				SpawnFirework(player);
+				SoundEngine.PlaySound(SoundID.Thunder);
+			}
+		}
+
+		public override void OnEndPose(Player player) {
+			for (int i = 0; i < 3; i++) {
+				SpawnFirework(player);
+			}
+		}
+
+		private void SpawnFirework(Player player) {
+			if (player.whoAmI != Main.myPlayer) {
+				return;
+			}
+
+			int fireworkProjectile = ProjectileID.RocketFireworksBoxRed + Main.rand.Next(4);
+			Projectile.NewProjectile(player.GetSource_FromThis(), player.Top, new Vector2(Main.rand.NextFloat(-2, 2), -Main.rand.NextFloat(2f, 4f)).RotatedByRandom(0.3f), fireworkProjectile, 0, 0, Main.myPlayer);
+		}
+	}
+
+	// This class doesn't autoload because it has a non-default constructor, it is loaded manually 2 times in VictoryPoseAdditionalLoader.Load
+	public class NonAutoloadVictoryPose : ModVictoryPose
+	{
+		private readonly string nameOverride;
+		private readonly int duration;
+
+		public override string Name => nameOverride;
+
+		public override string Texture => "ExampleMod/Content/Items/ExampleTooltipsItem"; // A texture with 4 smileys
+
+		public NonAutoloadVictoryPose(string name, int duration) {
+			this.nameOverride = name;
+			this.duration = duration;
+		}
+
+		public override void SetStaticDefaults() {
+			PoseTime = duration;
+
+			VictoryPoseID.Sets.NonBoss[Type] = true;
+		}
+
+		public override Rectangle? GetTextureFrame(Asset<Texture2D> texture) {
+			// Randomly choose one of the 4 faces in the texture.
+			return texture.Frame(1, 4, 0, Main.rand.Next(4));
+		}
+
+		public override void OnStartPose(Player player) {
+			if (player.whoAmI == Main.myPlayer) {
+				Main.blockMouse = true;
+			}
+		}
+
+		public override void Update(Player player) {
+			if (player.itemAnimation == 0) {
+				player.bodyFrame.Y = player.bodyFrame.Height; // 1 Hand up
+			}
+		}
+	}
+
+	// A slightly tweaked CreativeSacrificeParticle. IParticle are similar to Dust, but can be customized completely. 
+	public class PoseIconParticle : IParticle
```

Comment:
...except this one, it could be an inner class, or just a companion class to `VictoryPosePlayer`